### DISCUSSION TITLE
How about we destroy rtk-query?

### DISCUSSION THREAD

**Discussion by chrispcode** [redux-saga/redux-saga#2242] [Ideas]
<img width="648" alt="image" src="https://user-images.githubusercontent.com/7029482/149430038-e2c3a663-0fee-405c-ae98-6b177664ddd2.png">

**Comment by Andarist** [comment]
There is no intention to destroy any other library on our side 😉 If you believe that there is a place for some kind of `createSagaApi` API then the best thing to move this forward would be to draft a PR showcasing how this would be useful etc

**Comment by neurosnap** [comment]
I built an experimental library for my vision of how `redux-saga` could reach some feature parity with `rtk-query` and `react-query`: https://github.com/neurosnap/saga-query

Check it out and let me know what you think.
